libs/ml/examples/word2vec_embeddings_extraction/main.cpp

```diff
@@ -0,0 +1,79 @@
+//------------------------------------------------------------------------------
+//
+//   Copyright 2018-2019 Fetch.AI Limited
+//
+//   Licensed under the Apache License, Version 2.0 (the "License");
+//   you may not use this file except in compliance with the License.
+//   You may obtain a copy of the License at
+//
+//       http://www.apache.org/licenses/LICENSE-2.0
+//
+//   Unless required by applicable law or agreed to in writing, software
+//   distributed under the License is distributed on an "AS IS" BASIS,
+//   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//   See the License for the specific language governing permissions and
+//   limitations under the License.
+//
+//------------------------------------------------------------------------------
+
+#include "core/commandline/parameter_parser.hpp"
+#include "math/tensor.hpp"
+#include "ml/dataloaders/word2vec_loaders/sgns_w2v_dataloader.hpp"
+#include "ml/exceptions/exceptions.hpp"
+#include "ml/utilities/word2vec_utilities.hpp"
+
+#include <string>
+
+using namespace fetch::ml;
+using namespace fetch::ml::dataloaders;
+
+using DataType   = float;
+using TensorType = fetch::math::Tensor<DataType>;
+using SizeType   = fetch::math::SizeType;
+
```

Comment:
add helper function that takes in word as string and outputs vector from embedding matrix